typescript/sdk/src/providers/SmartProvider/SmartProvider.ts

```diff
@@ -97,6 +98,41 @@ export class HyperlaneSmartProvider
     this.supportedMethods = [...supportedMethods.values()];
   }
 
+  async getPriorityFee() {
+    try {
+      return BigNumber.from(await this.perform('maxPriorityFeePerGas', {}));
+    } catch (error) {
+      return BigNumber.from('1500000000');
+    }
+  }
+
+  async getFeeData(): Promise<providers.FeeData> {
+    // override hardcoded getFeedata
+    const { block, gasPrice } = await resolveProperties({
+      block: this.getBlock('latest'),
+      gasPrice: this.getGasPrice().catch((error) => {
```

Comment:
Are you certain we want to swallow errors here?

Reply:
Copied from the upstream implementation and would keep it as is to avoid deviations